LICENSE.md

```diff
@@ -0,0 +1,12 @@
+## Licenses
```

Comment:
```suggestion
<!--
SPDX-FileCopyrightText: 2022 Stephan Druskat
SPDX-License-Identifier: CC0-1.0
-->

## Licenses
```

Reply:
I think `reuse` wants this file to be called just `LICENSE`, at least it does not complain about a file with name without a license header.

Reply:
Ha, probably not, just forgot to add this last one. Will fix in the next commit.

Reply:
It does not contain license text though. What @led02 suggest is the way to go.

Reply:
> I think `reuse` wants this file to be called just `LICENSE`, at least it does not complain about a file with name without a license header.

This isn't actually in the REUSE specs AFAIK, more of a concession to GitHub.

GitHub also has an undocumented mechanism in that if you supply license texts in files called `LICENSE-APACHE` etc., they'll compile a list of licenses found for the license in the overview panel, see, e.g., https://github.com/rust-lang/rust-clippy. Unfortunately, I don't think this is compatible with REUSE...